backend/course/crawler.py

```diff
@@ -36,8 +42,14 @@ def get_subareas(self, year, semester, area):
         return list(filter(lambda s: s['code'] != '', subareas))
 
     def get_courses(self, year, semester, area, subarea):
+        logging.debug('Crawling %s-%s %s-%s' % (
+                      str(year), self.semester_name[semester],
+                      '(%s, %s)' % (area['code'], area['name']),
+                      '(%s, %s)' % (subarea['code'], subarea['name'])
+                      ))
```

Comment:
let's replace this code with new-style formatting from

```
logging.debug('Crawling %s-%s %s-%s' % (
    str(year), self.semester_name[semester],
    '(%s, %s)' % (area['code'], area['name']),
    '(%s, %s)' % (subarea['code'], subarea['name'])
))
```

to

```
logging.debug('Crawling {year}-{semester} ({area_code}, {area_name})-({subarea_code}, {subarea_name})'.format(
                    year=year,
                    semester=self.semester_name[semester],
                    area_code=area['code'],
                    area_name=area['name'],
                    subarea_code=subarea['code'],
                    subarea_name=subarea['name'],
    ))
```

like this.